app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt

```diff
@@ -63,7 +67,7 @@ private infix fun String.story(story: NodeData): Pair<String, NodeItem> {
 object ComponentRegistry {
 
   private val COMPONENTS_TREE = mapOf(
-    "Badge" story NodeData { Story of R.layout.fragment_badge },
+    "Badge" story NodeData({ Story of R.layout.fragment_badge }, "all"),
```

Comment:
This might not be the best approach, I'm thinking about leaving all info about this in the test case